### PR TITLE
Fix CI on Windows

### PR DESCRIPTION
**Pull request by rafzi**
This wouldn't really have to be in the translate function, but the constructor is too early to have the library path additions from the Arch plugins.

### CODE REVIEW

File: JITImpl/TCC/TCCJIT.cpp [hunk 1]
```diff
@@ -143,6 +141,8 @@ void *TCCJIT::translate(std::string code, std::set<std::string> headerpaths, std
     {
         tcc_add_symbol(s, sym.first.c_str(), sym.second);
     }
+    for (const auto &jitPath : etiss::jitExtLibPaths())
+        etiss::LibraryInterface::AddSearchPath(etiss::jitFiles() + jitPath);
```
Comment (wysiwyng) [inline JITImpl/TCC/TCCJIT.cpp:145]: maybe use braces here to keep consistent with coding style

File: include/etiss/LibraryInterface.h [hunk 2]
```diff
@@ -141,6 +141,8 @@ class LibraryInterface
 
     static unsigned getCurrentLibraryVersion();
 
+    static void AddSearchPath(const std::string &path);
```
Comment (wysiwyng) [inline include/etiss/LibraryInterface.h:144]: begin function name lower case to keep consistent

File: src/LibraryInterface.cpp [hunk 3]
```diff
@@ -601,3 +601,17 @@ std::shared_ptr<LibraryInterface> LibraryInterface::openSharedLibrary(std::strin
     return nullptr;
 #endif
 }
+
+
+void LibraryInterface::AddSearchPath(const std::string &path)
```
Comment (wysiwyng) [inline src/LibraryInterface.cpp:606]: see comment in LibraryInterface.h

**Review by wysiwyng — COMMENTED** [review]
Minor code style comments, looks good so far.

**Comment by JoGei** [comment]
Seems to fix Issue #65. Is there another way to tell TCC on Windows to link similar to '-rpath' in any GCC?
It seems to work without  https://github.com/tum-ei-eda/etiss/blob/0b62f0c1f68cd20fbf44c8a7064a36412b1f4aff/JITImpl/TCC/TCCJIT.cpp#L145

for ETISS on Linux?

**Comment by rafzi** [comment]
@wysiwyng thanks!

@JoGei no, TCC does not have this feature as far as i can tell. the call only happens on windows (HAS_GETPROC macro) and the function is a no-op on linux anyways.

**Pull request merged**